src/explore-education-statistics-common/src/modules/release/components/OfficialStatisticsSection.tsx

```diff
@@ -33,6 +33,26 @@ export default function OfficialStatisticsSection({
         </a>
         .
       </p>
+      <p>
+        Our statistical practice is regulated by the Office for Statistics
+        Regulation (OSR).
+      </p>
+      <p>
+        OSR sets the standards of trustworthiness, quality and value in the{' '}
+        <a href="https://code.statisticsauthority.gov.uk/the-code/">
```

Comment:
Should this be using a `target="_blank"`?

Reply:
Actually you could use the existing Link component.  I think it'll give you this for free.

Reply:
As discussed on Slack, we have two <Link> components but they each exist in Frontend and Admin respectively, so aren't available to Common. On balance we decided to leave this as-is, given that it's consistent with the other anchor tags higher up in this component.